src/table.jl

```diff
@@ -63,12 +65,14 @@ function Base.iterate(t::Table, st = 1)
 end
 
 Base.getproperty(row::Row, name::Symbol) = getproperty(getfield(row, :record), name)
+Base.getproperty(row::Row, name::Symbol) = getproperty(getfield(row, :geometry), name)
 Base.getproperty(t::Table, name::Symbol) = getproperty(getdbf(t), name)
 
 Base.propertynames(row::Row) = propertynames(getfield(row, :record))
 Base.propertynames(t::Table) = propertynames(getdbf(t))
 
 Tables.schema(t::Table) = Tables.schema(getdbf(t))
+Tables.schema(t::Row) = Tables.schema(getdbf(t))
```

Comment:
Is it normal to define a schema on a row? I thought this function was only meant for objects implementing the table interface, and Row by itself doesn't implement that.

Reply:
Right! To be honest, I have not worked with `Table.schema` objects before so I wasn't sure what the best approach would be here. Do you happen to know of anything I could look at to help me in figuring this out? Thanks!

Reply:
Here is the schema for the DBF part:
https://github.com/JuliaData/DBFTables.jl/blob/6b7ff043f9d5d9df0408006b7c612b9555a7315b/src/DBFTables.jl#L282-L288
So we should probably unpack that schema, and add geometry to the names, and the eltype of the shapes to the types.

Reply:
Ok, I think I was able to successfully do this @visr. Adding a new commit now.